static_src/test/unit/stores/user_store.spec.js

```diff
@@ -173,6 +173,37 @@ describe('UserStore', function () {
     });
   });
 
+  describe('on org user association received', function() {
+    it('should emit a change event if data changed', function() {
+      var spy = sandbox.spy(UserStore, 'emitChange');
+
+      AppDispatcher.handleViewAction({
```

Comment:
Is there a reason to not call the userOrgAssociationReceived action here rather then dispatching an event? It might be easier and more concise.